model_overview_simple_network_sim.md

```diff
@@ -0,0 +1,89 @@
+
+In the ***nodes-as-populations*** version, each node contains an age-structured population and fractional people within that population are recorded as in various disease states.  We assume that the scale of the simulation is such that individuals do not progress between age classes.   As we will see in the next section, disease progression in this version is largely deterministic.  This version is currently considered the core version of this model, and is likely to be usable at larger population scales. 
+
+In the ***nodes-as-individuals*** version, each node is a single infectious unit or individual, and can only have one disease state.  This version is far more stochastic, is currently considered an alternate version, and is likely to be more appropriate at finer scale. 
+
+**Disease progression compartmental model**
+We use a compartmental model including the states:
+* S: susceptible
+* E: exposed, but not yet infectious
+* A: asymptomatic infectious individuals
+* I: symptomatic infectious individuals
+* H: hospitalised individuals 
+* R: recovered (assumed immune)
+* D: dead
+
+Movement between **S** and **E** is mediated by the infectious process model, all other transitions are determined by an input transition matrix that encodes the rates along the arrows in the following diagram:
+
+![](https://i.imgur.com/2rrJ6Qm.png)
+
+Different rates by age class are possible. 
+
+In the *nodes-as-populations* case, movements between the compartments is deterministic, and floating-point numbers of people are recorded.
+
+In the *nodes-as-individuals* case, a state is chosen for the node using probabilities derived from the transition rates, and a nodes has exactly one state at any time.  
+
+**Infectious process model**
+The infectious process moves people from the **S**usceptible to the **E**xposed states/compartments. 
+
+In the *nodes-as-populations* model, there are two types of infection: within-node infection and between-node infection. Within-node infection uses an age-mixing matrix and the number of people within a node in an infecting state (currently only **A** and **I** - infection by hospitalised individuals in discussion) of each age to generate a number of new infections within age classes.  Between-node infection (currently only inflicted on adult people) is determined by the proportion of infecting individuals in a source node, the weight of the contact between nodes, and the proportion of susceptible individuals in the receiving node. 
+
+In the *nodes-as-individuals* model, all infection is between-nodes, and all infectious nodes transmit infection over edges to their susceptible neighbours with probability determined by the weight of the edge. 
+
+**Order of simulation**
+Loading input files is now separated from the basic simulation.  After initial setup of the network, population, and epidemiological parameters, the basic simulation runs in the following order:
```

Comment:
For what I understood from the code the order shouldn't really matter, the only two steps that change the same data is the infection within node and infection between nodes and it doesn't seem to matter which one runs first, right?

Reply:
Shouldn't, since we make changes to the next time step rathr than this one, but thought I'd include it anyway.